styles/modules/_userPage.scss

```diff
@@ -78,5 +60,152 @@
+        margin-right: $padMd;
+
+        & > * {
+          margin-bottom: $pad;
+
+          &:last-child {
+            margin-bottom: 0;
+          }
+        }
+
+        .categoryFilter {
+          max-height: 250px;
+          overflow: auto;
+
+          .moreCatsWrap {
+            .moreCats, .showLess {
+              display: none;
+            }
+
+            &.expanded {
+              .moreCats, .showLess {
+                display: block;
+              }
+
+              .showMore {
+                display: none;
+              }
+            }
+          }
+        }
+      }
+
+      .storeListings {
+        flex-shrink: 1;
+        width: 100%;
+
+        .listingsWrap {
+          flex-wrap: wrap;
+
+          &:empty {
+            & ~ .noResults {
+              display: block;
+            }
+          }
+
+          .col {
+            margin: 0 $padMd $padMd 0;
+            width: 225px;
+            height: 283px;
+            border-style: solid;
+            border-width: 1px;
+            font-size: $tx5b;
+            cursor: pointer;
+            position: relative;
+
+            &:nth-child(3n) {
+              margin-right: 0;
+            }
+
+            .listingImage {
+              width: 223px;
+              height: 225px;
+              position: relative;
+
+              .editOverlay {
+                display: none;
+                position: absolute;
+                top: 0;
+                left: 0;
+                right: 0;
+                bottom: 0;
+                align-items: center;
+                justify-content: center;
+                z-index: 1; // so it's above the overlay icons                
+
+                .overlayPanel {
+                  // Using seperate element just for the overlay so that
+                  // the opacity doesn't affect the buttons, but we could 
+                  // still use the clrS class (instead of RGBA).
+                  opacity: 0.9;
+                  position: absolute;
+                  top: 0;
+                  left: 0;
+                  right: 0;
+                  bottom: 0;
+                }
+
+                .btn {
+                  z-index: 1;
+                  border-left-width: 0;
+
+                  &:first-of-type {
+                    border-left-width: 1px;
+                  }
+                }
+              }
+            }
+
+            &:hover {
+              .editOverlay {
+                display: flex;
+              }
+
+              &.ownListing {
+                // todo: change to focused color variable
+                border-color: black;
+              }
+            }
+
+            .listingIcons {
+              position: absolute;
+              top: 8px;
+              right: 8px;
+            }
+          }
+        }
+
+        .noResults {
+          display: none;
+        }
+
+        .listingsHeaderRow {
+          .listingsCount {
+            flex-basis: 100%;
```

Comment:
This probably isn't the best way to do this.

It would be better to give this flex-grow: 1, and then wrap the next sibling div in a div, and remove the floR class from it.

The reason the sibling needed floR to collapse it is you can't have an element with display:flex as a direct child of a flex element, or it mucks up how the space is distributed.